onnxruntime/core/session/provider_bridge_ort.cc

```diff
@@ -1800,8 +1801,21 @@ std::shared_ptr<IExecutionProviderFactory> OpenVINOProviderFactoryCreator::Creat
   return s_library_openvino.Get().CreateExecutionProviderFactory(&ov_options_converted_map);
 }
 
-std::shared_ptr<IExecutionProviderFactory> OpenVINOProviderFactoryCreator::Create(const ProviderOptions* provider_options_map) {
-  return s_library_openvino.Get().CreateExecutionProviderFactory(provider_options_map);
+ProviderOptions ORTSessionOptionsToOrtOpenVINOProviderOptions(const ProviderOptions ov_options, const SessionOptions *session_options) {
+  ProviderOptions so_ov_options_map(ov_options);
+  if(session_options){
+    bool disable_cpu_fallback = session_options->config_options.GetConfigOrDefault(
+                                   kOrtSessionOptionsDisableCPUEPFallback, "0") == "1";
```

Comment:
Will it be 0 or 1 only ? Can it be True of False

Reply:
The API GetConfigOrDefault(KEY, value) expects the default value. For CPUEPFallback, the flag should be set to 1 for disabling fallback